src/Node/Expression/Variable.php

```diff
@@ -24,7 +24,7 @@ class Variable extends Expression {
     public function getName() {
         if (
             $this->name instanceof Token &&
-            $name = substr($this->name->getText($this->getFileContents()), 1)
+            $name = ltrim($this->name->getText($this->getFileContents()), '$')
```

Comment:
So this trims all `$` from the left - what about variable variables? e.g. `$$var`